plugins/active_directory_ldap/komand_active_directory_ldap/actions/disable_users/action.py

```diff
@@ -22,10 +22,8 @@ def run(self, params={}):
             )
 
         disabled_users = self.connection.client.manage_users(params.get(Input.DISTINGUISHED_NAMES), False)
-        success = not bool(disabled_users.get("failures"))
 
         return {
-            Output.ALL_OPERATIONS_SUCCEEDED: success,
-            Output.SUCCESSFUL_DISABLEMENTS: disabled_users.get("successes"),
-            Output.UNSUCCESSFUL_DISABLEMENTS: disabled_users.get("failures"),
+            Output.COMPLETED: disabled_users.get("successes"),
+            Output.FAILED: disabled_users.get("failures"),
```

Comment:
```suggestion
            Output.COMPLETED: disabled_users.get("successes", []),
            Output.FAILED: disabled_users.get("failures", []),
```

Reply:
This should be fine as the manage users returns empty arrays if there are no contents in either.